compose.yml

```diff
@@ -85,6 +85,26 @@ services:
         max-file: "5"
     restart: on-failure
 
+  sdx:
+    image: usdotjpoode/jpo-sdw-depositor:2024-q3
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      SDW_DESTINATION_URL: ${SDW_DESTINATION_URL}
+      SDW_API_KEY: ${SDW_API_KEY}
+      SDW_EMAIL_LIST: ${SDW_EMAIL_LIST}
+      SDW_EMAIL_FROM: ${SDW_EMAIL_FROM}
+      SPRING_MAIL_HOST: ${SPRING_MAIL_HOST}
+      SPRING_MAIL_PORT: ${SPRING_MAIL_PORT}
+    depends_on:
+      - ode
+    volumes:
+      - ${DOCKER_SHARED_VOLUME}:/asn1_codec_share
```

Comment:
This volume mount probably isn't necessary.

Reply:
Removed